src/OrleansRuntime/GrainTypeManager/GrainTypeManager.cs

```diff
@@ -153,7 +155,15 @@ private void InitializeGrainClassData(SiloAssemblyLoader loader, bool strict)
 
         private void InitializeInvokerMap(SiloAssemblyLoader loader, bool strict)
         {
-            IEnumerable<KeyValuePair<int, Type>> types = loader.GetGrainMethodInvokerTypes(strict);
+            // Load all existing implementations.
+            var existing = new HashSet<int>(loader.GetGrainMethodInvokerTypes(strict).Select(_ => _.Key));
+            
+            // Create implementations for all types which do not already have an implementation.
+            GrainMethodInvokerGenerator.CreateForCurrentlyLoadedAssemblies(type => !existing.Contains(type));
```

Comment:
I may be better to call `GrainMethodInvokerGenerator.CreateForCurrentlyLoadedAssemblies()` and `GrainReferenceGenerator.CreateForCurrentlyLoadedAssemblies()` from the same point in the silo initialization point for better clarity about when they get invoked. Is there a reason to invoke them separately instead of wrapping the two calls in a single public/internal method that does both like Bootstrap class does?

In the current form, the sequence of call isn't immediately clear to me. I can figure it out, but I thought it would be more maintainable if we could make one call to the code generator from a single point in the silo initialization process, unless there are difficulties with that.